cpp/src/gandiva/regex_functions_holder.cc

```diff
@@ -17,15 +17,12 @@
 
 #include "gandiva/regex_functions_holder.h"
 #include <regex>
+#include "arrow/util/macros.h"
```

Comment:
I am not very familiar with the gandiva code base.  Is it ok/normal to use Arrow macros here?  I assume so.

Reply:
Yep. Gandiva uses various headers in the arrow/util directory.